### ISSUE TITLE
style handling

### ISSUE THREAD

**Issue by blynx** [felixwahner/TextformatterOEmbed#8]
Hi,

I think it would be nice to turn off the automatic style inclusion so one can put the styles outside of the markup. Maybe a checkbox in the module settings.

Also on line 208, I think `_inner--` should actually be `__inner--` with double underscore?

cheers
Steffen

**Comment by gmclelland** [comment]
I agree.  I noticed this when I viewed one of my website's teaser listing.  It was showing the inline style code before my teaser.  I was able to filter it out with Twig, but it would be nice to not automatically add these styles.

```twig
<div class="post-content">{{ post.event_description|striptags('<style>')|truncate(440, false, "...") }}</div>
```

What about adding the styles once to the end of the document's head tag, that way it's not repeating these styles for every embed?  That could be the default option that comes checked.  That way existing installs would keep working without problems, but other site builders could also disable this option and include their own css if needed.
